service/cart_http.go

```diff
@@ -0,0 +1,195 @@
+package service
+
+import (
+	"strings"
+	"strconv"
+	// "encoding/json"
+	"net/http"
+	logger "github.com/sirupsen/logrus"
+)
+
+// type successResponse struct {
```

Comment:
why this code is commented?

Reply:
I had created and used it earlier but later on saw that same functionality is written in responce.go so commented it to check whether it will work or not.

Reply:
Removed it